### DISCUSSION TITLE
Is there a way to not display then entire webMAN games folder in XMB?

### DISCUSSION THREAD

**Discussion by ghost** [aldostools/webMAN-MOD#548] [Q&A] [answered]
For example, if someone wanted to solely mount their games through the web server with an external device, or solely use the sMAN GUI, is it possible to not display the webMAN folder and games? Would there be any benefit in doing this (RAM, CPU, etc)?

I tried looking in the setup section, but I was never able to find an option which would hide the games in XMB.

**Comment by aldostools** [comment, marked as answer]
You can edit category_games.xml and remove or comment out the query "xmb_app3 or edit the source src="xmb://localhost/dev_hdd0/xmlhost/game_plugin/fb.xml#seg_fb"

Or you can disable the scan for content at startup and delete /dev_hdd0/xmlhost/game_plugin/fb.xml

**Comment by ghost** [comment]
Thanks for the help Aldo. Hopefully an option to do this automatically can be added to wMM setup in the future.
